bigtable-hbase-dataflow/src/test/java/com/google/cloud/bigtable/dataflow/CloudBigtableIOTest.java

```diff
@@ -81,8 +70,6 @@ public void testInitialize() throws Exception{
     checkRegistry(Put.class);
```

Comment:
Can you please move this into the Integration test?  That seems like a better fit.